### ISSUE TITLE
write python code, which load raw data from kaggle and make preprocessing using Imputer , Standarization and Normalization then upload the code on your githup

### ISSUE THREAD

**Issue by Elgohary10** [Elgohary10/Ali-Elgohary-#1]
import os
import pandas as pd
from sklearn.impute import SimpleImputer
from sklearn.preprocessing import StandardScaler, MinMaxScaler

# Set up Kaggle API credentials
os.environ['KAGGLE_USERNAME'] = 'your_kaggle_username'
os.environ['KAGGLE_KEY'] = 'your_kaggle_key'

# Download dataset from Kaggle
!kaggle datasets download -d <dataset-name> --unzip

# Load the dataset
data = pd.read_csv('<path-to-your-dataset>')

# Display the first few rows of the dataset
print(data.head())

# Imputation
imputer = SimpleImputer(strategy='mean')
data_imputed = imputer.fit_transform(data)

# Standardization
scaler = StandardScaler()
data_standardized = scaler.fit_transform(data_imputed)

# Normalization
normalizer = MinMaxScaler()
data_normalized = normalizer.fit_transform(data_standardized)

# Convert back to DataFrame
data_processed = pd.DataFrame(data_normalized, columns=data.columns)

# Display the first few rows of the processed dataset
print(data_processed.head())

# Save the processed dataset to a new CSV file
data_processed.to_csv('processed_data.csv', index=False)
